subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -769,7 +769,6 @@ int ull_conn_llcp(struct ll_conn *conn, u32_t ticks_at_expire, u16_t lazy)
 {
 	LL_ASSERT(conn->lll.handle != 0xFFFF);
 
-#if defined(CONFIG_BT_CTLR_CONN_PARAM_REQ) || defined(CONFIG_BT_CTLR_PHY)
```

Comment:
Why removing the configuration parameter instead of changing the Vega vendor to select it? For the Vega, this parameter is disabled.

Name: BT_CTLR_CONN_PARAM_REQ_SUPPORT
Type: bool
Value: n

Direct dependencies (=y):
     BT_HCI(=y)
  && BT(=y)

Symbols currently n-selecting this symbol (no effect):
  - BT_LL_SW_LEGACY
  - BT_LLL_VENDOR_NORDIC

Reply:
The check for conn update request was put under conn param req #ifdef which are unrelated